bin/employee_permissions/tool_permission/github_employee_permission.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../../../lib/cdo/honeybadger'
+require_relative '../../employee_permissions/tool_permission/code_org_employee_permission'
+class GithubEmployeePermission < CodeOrgEmployeePermission
+  @@github_token = ''
+  @@org_name = 'code-dot-org'
+
+  def self.get_teams
+    teams_curl_response = `curl -s \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: Bearer #{@@github_token}" \
+      https://api.github.com/orgs/#{@@org_name}/teams`
+    JSON.parse teams_curl_response
+  end
+
+  def self.get_team_members(team_slug)
+    teams_members_curl_response = `curl -s \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: Bearer #{@@github_token}" \
+      https://api.github.com/orgs/#{@@org_name}/teams/#{team_slug}/members`
+    JSON.parse teams_members_curl_response
+  end
+
+  def self.get_role(team_slug, user_name)
+    role_curl_response = `curl -s \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: Bearer #{@@github_token}" \
+      https://api.github.com/orgs/#{@@org_name}/teams/#{team_slug}/memberships/#{user_name}`
+    parse_response = JSON.parse role_curl_response
+    parse_response['role']
+  end
+
+  def self.get_all_members_impl
+    members = []
+    teams = get_teams
+    teams.each do |team|
+      team_slug = team['slug']
+      team_members = get_team_members(team_slug)
+      team_members.each do |member|
+        member_user_name = member['login']
+        members << {
+          id: member['id'],
+          search_key: member_user_name,
+          role: get_role(team_slug, member_user_name),
+          user_name: member_user_name,
+          org_name: @@org_name,
+          team_name: team['name'],
+          team_slug: team['slug']
+        }
+      end
+    end
+    return members
```

Comment:
as above, this one method is using an explicit return whereas the rest of this class is using implicit returns. It doesn't really matter which way we go (although I'm personally a fan of "explicit is better than implicit"), but what does matter is that we're consistent. We should either make this an implicit return to match the other methods in this class, or update the rest of the methods to match this one.